### PR TITLE
The user is now redirected to the login page upon activation of their account.

### PR DESCRIPTION
**Pull request by nickwxo**
This branch simply implements the redirection of a user that has just activated their account via the email link to the login page.

### CODE REVIEW

File: routes/user/user.js [hunk 1]
```diff
@@ -189,7 +189,8 @@ exports.activateUser = function(req, res, next) {
                 //removing the token and password from the response (security)
                 docs.us_activation_token = undefined;
                 docs.us_password = undefined;
-                res.send(docs);
+                console.log(docs);
+                res.redirect("/#/signin");
```
Comment (matjames007) [inline routes/user/user.js]: @nick22891 No the redirect must not happen using the backend.  Instead use the front-end application to accept the token as an URL parameter within the app controller.

**Comment by nickwxo** [comment]
@matjames007 I've re-implemented as per your recommendations. This pull request can be reviewed again.

File: public/javascript/controllers/user-ctrl.js [hunk 2]
```diff
@@ -63,17 +63,28 @@ angular.module('harvestv2')
 
         }
     ]
-).controller("UserLoginCtrl", ['$scope', '$location', '$routeParams', 'UserFactory',
+).controller("UserLoginCtrl", ['$scope', '$location', '$routeParams', 'UserFactory', 'UserActivationFactory',
         'AuthenticationFactory', 'PlatformFactory',
-        function($scope, $location, $routeParams, UserFactory, AuthenticationFactory, PlatformFactory) {
+        function($scope, $location, $routeParams, UserFactory, UserActivationFactory, AuthenticationFactory, PlatformFactory) {
             var credentials = {};
```
Comment (matjames007) [inline public/javascript/controllers/user-ctrl.js:70]: @nick22891 please comment your code.

**Comment by nickwxo** [comment]
There were some merge conflicts that needed to be resolved and I pulled 'develop' into this branch. I've also added some comments as per @matjames007 recommendations.

Please review again when you can...

File: routes/user/user.js [hunk 3]
```diff
@@ -189,7 +189,7 @@ exports.createUser = function(req, res, next) {
                 "Hi " + user.us_user_first_name + ",\n" +
                 "You have successfully created an Harvest account. " +
                 "To complete the process, activate your account by clicking on the link below: " +
-                "http://localhost:3000/activate/" + user.us_activation_token + "\n" +
+                "http://localhost:3000/#/activate/" + user.us_activation_token + "\n" +
```
Comment (matjames007) [inline routes/user/user.js]: We need to automatically generate this URL... when it moves to the staging environment it will not work.

Comment (nickwxo) [reply routes/user/user.js]: @matjames007 Fixed.

**Pull request merged**